docs/local-development.md

```diff
@@ -0,0 +1,23 @@
+# Local Development
+
+## create-dao development
+
+1.  Run `yarn` to install packages
+2.  In one terminal
+3.  Run `yarn chain` for local ETH test accounts
+4.  In another terminal
+5.  Run `yarn deploy` to deploy the smart contract
+6.  Run `yarn dev` for next development
+
+## Workspaces
+
+This project uses workspaces to help keep ongoing development easier. All scripts can be run from the root directory without going into the the individual folders.
+
+### Adding more workspaces
+
+To add another workspace,
+
+1. In the package.json file in the root directory, add the folder that contains the new workspace package.json file to the workspaces array.
```

Comment:
If we update this instruction to tell the user to create the new workspace in the `packages` folder, they won't have to touch the root `package.json` at all since you're using the `packages/*` glob. I think that'd be preferable.